data-service/src/test/java/com/amazonaws/blox/dataservice/repository/EnvironmentRepositoryDDBTest.java

```diff
@@ -133,34 +192,32 @@ public void createEnvironment() throws StorageException, EnvironmentExistsExcept
 
     verify(dynamoDBMapper).save(environmentDDBRecordCaptor.capture());
     assertEquals(
-        environmentDDBRecord.getAttributes(),
+        environment.getInstanceGroup().getAttributes(),
```

Comment:
Does this work for non-empty attributes? AFAICT, `InstanceGroup::getAttributes` returns a `Set<Attribute>`, but `EnvironmentDDBRecord::getAttributes` returns a `Set<String>`.

Reply:
Yes there's a TODO in the code to add and map attributes

Reply:
I just ask, because this is also throwing an `unchecked assignment` warning in the build output.